conf.json

```diff
@@ -1,9 +1,9 @@
-{ settings: 
+{ "settings": 
 	[
 		{
-		  "src_table": "info",
-		  "dest_table": "info",
-		  "column_mapping": {"ID_": "id", "DURATION_": "start_time"},
+		  "src_table": "Camunda_dump_example_csv",
```

Comment:
@VincenzoFerme Yeah, I was using the dump you gave me as an example to test, but it's meant to be the name of a table.